src/EnrollOnPremController.js

```diff
@@ -72,6 +72,18 @@ function (Okta, FormType, FormController, Footer, TextBox) {
         noButtonBar: true,
         autoSave: true,
         className: getClassName(provider),
+        initialize: function() {
+          if (isRSA(provider)) {
+            this.listenTo(this.model, 'error', _.bind(function (source, error) {
+              if (error && error.status === 409) {
+                // 409 means we are in change pin, so we should clear out answer input
+                this.model.set('passCode', '');
+                this.$('.o-form-input-name-passCode input').val('');
```

Comment:
couldn't we trigger 'change' and that would update the model as well?

Reply:
would have to be `change` in the input right? I don't have the input here, and probably it is simpler to set the value as it is, as opposed to find the input. Does that sounds right?